torch/csrc/distributed/rpc/rref_impl.cpp

```diff
@@ -274,13 +265,7 @@ const IValue& OwnerRRef::getValue() const {
       !getTimedOut(),
       "RRef creation via rpc.remote() timed out, and it "
       "is possible that the RRef on the owner node does not exist.");
-  future_->wait();
-  if (future_->hasError()) {
-    (void)future_->value(); // Throws the error.
-  }
-  // Before accessing the value in this RRef, current CUDA streams must wait
-  // for pending CUDA operations that create the value.
-  blockCurrentStreams(events_);
```

Comment:
`blockCurrentStreams` enqueues a wait for the RRef's events on each of the current streams, but just above we call `wait` on the future which does that too.